### ISSUE TITLE
Support inflection of verbs

### ISSUE THREAD

**Issue by grhoten** [unicode-org/inflection#11]
We should be able to inflect verbs. For example, in English, I may have the phrase "The ${device} is on". The word "is" is a verb, and can have 2 forms in English in the context of that sentence. It can be "is" or "are" in English. In some other languages, they can also vary depending on the grammatical gender of the noun being referenced.

As a discussion point, we can also consider changing the tense of a verb, like changing "is" (present tense) to "was" (past tense). Changing the tense in a sentence is a rare grammatical category to change, but it can be helpful to support.

**Comment by macchiati** [comment]
I agree that that is in scope. 
(FYI, Not needed for MF, because that uses selection of variants based on plural category.)

**Comment by grhoten** [comment]
You're correct when your message formatting architecture is based on conditionals. We used to have that for one of our frameworks, but we have been switching towards implicit/derived constraints. We've found that conditionals can be helpful, but it's much easier to translate and author if you can take a span of text and just say "make the words in the phrase all agree with the lone object being referenced".

The heavy use of conditionals/selectors was a design point that I disagreed on in the MFWG. It's easier to implement. It's also more labor intensive for authors and translators due to how long it takes to write and modify conditions. Translators aren't programmers, and the conditional logic adds to the learning curve and repetitive code copying. In contrast, the [automatic grammar agreement in Foundation](https://www.youtube.com/watch?v=C2e7hYIkqoM) has no conditionals, but it does support constraints.

**Comment by nciric** [comment]
The plural/singular change is one example for verb inflection.

Another shows up in Slavic languages, for verbs in past tense have to match the gender and number of the subject (he/she/it worked):

```
Past, singular:

Ona je radila  // je is auxiliary verb and stays the same in singular , Ona is a pronoun/subject
On je radio
Ono je radilo

Past, plural:

One su radile  // Even the auxiliary verb changes to match the plural/singular.
Oni su radili
Ona su radila
```
but not in present tense (he/she/it works)

```
Present:

On/ona/ono radi
```

**Comment by grhoten** [comment]
I agree. Several of the languages have gendered verbs.

**Comment by grhoten** [comment]
I'd like to nominate this to be resolved with pull request #35.
